### DISCUSSION TITLE
Introductions - please introduce yourself here

### DISCUSSION THREAD

**Discussion by shayan-cyber** [CiviWiki/OpenCiviWiki#995] [General]
This thread is intended for new community members to introduce themselves. Please add a comment with a brief introduction. We can then help you get started contributing to CiviWiki :smile: 

## Original introduction made by @shayan-cyber (we've repurposed this thread)

Hey everyone. I'm starting this discussion thread to introduce myself.

I'm **Shayan Debroy** ,  pursuing B.Tech in Electronics and Communication Engineering from National Institute Of Technology, Silchar(India). I taught myself **Python and django** and came towards web development only because of the quality resources made available by the **open source community** .
It is my turn to help the open source community. I want to start my open source journey with this wonderful project **CiviWiki**.
I'm inclined to learn new technologies and have good experience in **web development with django.**
Eagerly waiting to learn as well as help the open source community.

**Comment by brylie** [comment]
Welcome @shayan-cyber! Thanks for your interest in helping the CiviWiki project.

We are currently in the most need of testers to go through the software and find bugs. There are a lot of code changes ongoing, so we may have introduced some bugs. Would you be interested in running the software locally and testing all of the features?

**Reply by shayan-cyber** [reply]
yeah I can do that

**Comment by wassafshahzad** [comment]
I am a little late for an introduction but here we go.
My name is Wassaf Shahzad and I amd currently working as Software Engineer . I have +2 years of experience and can work in python, java and Javascript.

**Reply by brylie** [reply]
Welcome, @wassafshahzad! We really appreciate your contributions so far :smiley:

**Reply by wassafshahzad** [reply]
Thank you @brylie for the opportunity

**Comment by adarshjhaa100** [comment]
Hello I am Adarsh Jha, CS graduate from India. I am experienced in Django, Spring Boot, react, angular although open to other technologies that may improve me as a full stack developer.  
I am new to open source projects and hope will be able to help and learn a lot from y' all.

**Reply by brylie** [reply]
Welcome @adarshjhaa100!

Would you like to help out by testing the CiviWiki project? If so, you can start a development environment, as outlined in our contributing guide, and go through the app to see how things work. If you encounter any issues, feel free to open a discussion or bug report. Any feature ideas are also welcome! :smile:

**Comment by adarshjhaa100** [comment]
Yes I would like to help out testing the project.

**Reply by brylie** [reply]
Ok, great! Please follow the instructions under the [development section of our contributing guide](https://github.com/CiviWiki/OpenCiviWiki/blob/develop/CONTRIBUTING.md#development). Once the project is running, you can register a local user and start clicking around :smiley:

**Comment by AbhijithGanesh** [comment]
Hello everyone! I am Abhijith Ganesh, a freshman engineering student from India. sorry for the late intro😅

**Reply by brylie** [reply]
Welcome @AbhijithGanesh! You've made excellent contributions!

**Reply by AbhijithGanesh** [reply]
Thanks for those extremely kind words Brylie

**Comment by vishnudas-bluefox** [comment]
Am Vishnu An Engineering studentfrom india :)

**Reply by brylie** [reply]
Welcome to the CiviWiki Community 😃

**Comment by anirudhgray** [comment]
Hey! I'm a freshman engineering student with some experience in Python (and Django!), but close to none in contributing to open source projects... so I'm looking forward to helping out where I can and learning while at it 😄

**Reply by brylie** [reply]
Welcome! We can use help bug hunting right now.

**Comment by Vinit-Kumar-Shah** [comment]
Hello Everyone! I am Vinit. I'm a senior IT Engineering student. I have experience in working with Web Applications belonging to different kinds of tech stacks. I can really be helpful with managing and debugging Python (Django) related issues here.
I am new in this community but I really look forward to contributing so please feel free to tag me in any issue you feel I can help with.😃

**Reply by brylie** [reply]
Welcome! We're glad to have you as a contributor 😃

Would you like to help us test the code to look for bugs? If so, just set up a development environment as outlined in our contributing guide.

**Reply by Vinit-Kumar-Shah** [reply]
Sure I'll have a look into it

**Comment by aman162000** [comment]
I'm Aman , pursuing B.E in Computer Science and Engineering. I'm self-taught Python and Django developer. I really look forward to contribute.

**Reply by brylie** [reply]
Welcome @aman162000! We look forward to your contributions and appreciate your interest in CiviWiki :smiley:

**Comment by jchenwth101** [comment]
Hello,
I'm Joel.  I'm undertaking an open source class in a CS program.  I was hoping to find something/some ways to contribute.  I have 3 + years of python, testing, continuous integration... As a small test, I added a couple of lines to a "copy" of the CiviWiki privacy policy (section 2 part a).  This is my first attempt at contributing to any open source project.  This said, I'm interested in testing if there's a pressing issue..
[CiviWiki Privacy Policy copy.odt](https://github.com/CiviWiki/OpenCiviWiki/files/7406551/CiviWiki.Privacy.Policy.copy.odt)
.

**Reply by brylie** [reply]
Welcome @jchenwth101! We can use some help with testing and continuous integration. Currently though, we are working to port some Backbone.js templates over to Django template language/views. Would you be available to help us with this last-mile migration from the old "SPA"-style code to a more conventional/maintainable Django MTV architecture?

**Comment by vadini-agrawal** [comment]
Hi! I'm a Masters student in Computer Science looking to start contributing to open source / this project. I liked the mission for this project! I can start with downloading the repo & exploring a bit, but let me know if there are any specific aspects I can help out with. I'm most familiar with Python and HTML/CSS, but have worked with JavaScript as well. Excited to help out!

**Reply by brylie** [reply]
Welcome @vadini-agrawal. For now, we are glad to have beta testers to look for bugs in the code. We have some major revisions in progress, so catching bugs is very helpful.

**Comment by adityachoudary54** [comment]
Hello I am Aditya Choudhary, CS graduate from India. I currently have 6 months of experience working as a backend engineer and would love to acquire the skills in the field and contribute in this project. Looking forward to learn a lot from you guys 😁

**Comment by Jyotika2001** [comment]
Hello everyone, Jyotika here, a 3rd year CS student from Mumbai. I have a keen interest in Machine Learning and Data Science, with 2 years experience in Web Development as well. I am new to open source and am looking forward to learning a lot. I loved the mission of this project, as well as the catchy name! I have started exploring the web system a bit, but let me know if there is any specific bug I can help out with.

Thank you

**Comment by Green-Abe** [comment]
Hello everyone, hopefully I can contribute ✊😁

**Comment by xuro4** [comment]
Hi, I'm Robert, a senior at the University of Michigan studying CS. I have prior internship experience with Python, C++, and OCaml. I'm looking to get more involved with the open source community and thought CiviWiki was a cool place to start!

**Comment by fbarnea** [comment]
Hi everyone, I'm Florin, a junior developer with knowledge of python and Django. I work as a BI dev currently but looking to transition to a software eng role. I don't have any experience with open source software so I am excited to start helping on this project. 
I had a very similar idea years ago before I knew how to code (it was called Agora) so I'm very interested in helping this project succeed.

**Comment by Zeeshan19982000** [comment]
Hello I am Zeeshan Rafique, Mechatronics Engineer from Pakistan. My Skills included POWER BI and SQL.
I am new to open source projects and hope will be able to help and learn a lot from you all.

**Comment by abdz-128** [comment]
Hello,
I am a final year Electrical Engineering student specializing in Computer Engineering. I know Cpp, Python, Asm and am interested to work in beginner open source projects like Web dev, ML, DL and Data Analysis because I am a Noob in open source contribution. I would be honored to help contribute to your projects. Looking forward to help you out.
Peace out

**Comment by ksharma20** [comment]
Hello Foks,
I'm Keshav Sharma, currently pursuing 4th year of my Computer Engineering from Thapar University, Patiala.
I have been working in Python and Django for a while and Would like to contribute to our well-loved open source community of better future. I am new to open source so, I would really appreciate guidance and help from seniors 😁.

**Reply by AbhijithGanesh** [reply]
Nice to have you here Keshav
